samples/nested-pipeline/compose.py

```diff
@@ -12,96 +12,88 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-import kfp.dsl as dsl
-
-
-class GetFrequentWordOp(dsl.ContainerOp):
-  """A get frequent word class representing a component in ML Pipelines.
-
-  The class provides a nice interface to users by hiding details such as container,
-  command, arguments.
-  """
-  def __init__(self, name, message):
-    """Args:
-         name: An identifier of the step which needs to be unique within a pipeline.
-         message: a dsl.PipelineParam object representing an input message.
-    """
-    super(GetFrequentWordOp, self).__init__(
-        name=name,
-        image='python:3.5-jessie',
-        command=['bash', '-c'],
-        arguments=['echo \'%s\' | tee /tmp/results.txt ; python -c "from collections import Counter; '
-                  'foo=open(\'"/tmp/results.txt"\', \'r\').read();'
-                  'words = Counter(foo.split()); print(max(words, key=words.get))" '
-                  '| tee /tmp/message.txt' % message],
-        file_outputs={'word': '/tmp/message.txt'})
-
-
-class SaveMessageOp(dsl.ContainerOp):
-  """A class representing a component in ML Pipelines.
-
-  It saves a message to a given output_path.
-  """
-  def __init__(self, name, message, output_path):
-    """Args:
-         name: An identifier of the step which needs to be unique within a pipeline.
-         message: a dsl.PipelineParam object representing the message to be saved.
-         output_path: a dsl.PipelineParam object representing the GCS path for output file.
-    """
-    super(SaveMessageOp, self).__init__(
-        name=name,
-        image='google/cloud-sdk',
-        command=['sh', '-c'],
-        arguments=['echo \'%s\' | tee /tmp/results.txt | gsutil cp /tmp/results.txt \'%s\'' % (message, output_path)])
+from kfp import dsl
+from kfp import components
+
+def frequent_word(message: str) -> str:
+    """get frequent word."""
+    from collections import Counter
+    words = Counter(message.split())
+    result = max(words, key=words.get)
+    print(result)
+    return result
+
+frequent_word_op = components.create_component_from_func(
+    func=frequent_word, base_image='python:3.5-jessie')
+
+save_message_op = components.load_component_from_text("""
+    name: 'Save Message'
+    description: |
+      save message to a given output_path
+    inputs:
+      - {name: message, type: String, description: 'Required. message'}
+      - {name: output_path, type: String, description: 'Required. output path'}
+    implementation:
+      container:
+        image: google/cloud-sdk
+        command:
+        - sh
+        - -c
+        - |
+          set -e -x
```

Comment:
We don't need to print out the execution command since it's not part of the original component.
```suggestion
          set -e
```